synchrad/calc.py

```diff
@@ -85,8 +86,8 @@ def __init__(self, Args={}, file_spectrum=None):
 
         if mpi_installed:
             self.comm = MPI.COMM_WORLD
-            self.rank = self.comm.rank
-            self.size = self.comm.size
```

Comment:
The `mpi4py` documentation seems to recommend using `Get_rank()` and `Get_size()` here.

Reply:
looks fine to me